tst/EnergyPlus/unit/ChillerExhaustAbsorption.unit.cc

```diff
@@ -1155,3 +1156,280 @@ TEST_F(EnergyPlusFixture, ExhAbsorption_GetInput_Multiple_Objects_Test)
+        "    -3.24613,                !- Coefficient3 x**2                                                             ",
+        "    1.497306,                !- Coefficient4 x**3                                                             ",
+        "    0.03,                    !- Minimum Value of x                                                            ",
+        "    1.0;                     !- Maximum Value of x                                                            ",
+        "                                                                                                              ",
+        "  Curve:Biquadratic,                                                                                          ",
+        "    Capstone C65 Power_vs_Temp_Elev,  !- Name                                                                 ",
+        "    1.2027697,               !- Coefficient1 Constant                                                         ",
+        "    -9.671305E-03,           !- Coefficient2 x                                                                ",
+        "    -4.860793E-06,           !- Coefficient3 x**2                                                             ",
+        "    -1.542394E-04,           !- Coefficient4 y                                                                ",
+        "    9.111418E-09,            !- Coefficient5 y**2                                                             ",
+        "    8.797885E-07,            !- Coefficient6 x*y                                                              ",
+        "    -17.8,                   !- Minimum Value of x                                                            ",
+        "    50.0,                    !- Maximum Value of x                                                            ",
+        "    0.0,                     !- Minimum Value of y                                                            ",
+        "    3050.,                   !- Maximum Value of y                                                            ",
+        "    ,                        !- Minimum Curve Output                                                          ",
+        "    ,                        !- Maximum Curve Output                                                          ",
+        "    Temperature,             !- Input Unit Type for X                                                         ",
+        "    Distance,                !- Input Unit Type for Y                                                         ",
+        "    Dimensionless;           !- Output Unit Type                                                              ",
+
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    compare_err_stream("");
+
+    GetExhaustAbsorberInput(*state);
+
+    auto &thisChillerHeater = state->dataChillerExhaustAbsorption->ExhaustAbsorber(1);
+
+    Real64 loadinput = -5000.0;
+    bool runflaginput = true;
```

Comment:
This variable is unused.

Reply:
Thanks--will deal with while trying adding the "at end" error message check in unit test.